doc/river.1.scd

```diff
@@ -0,0 +1,35 @@
+RIVER(1) "github.com/ifreund/river" "General Commands Manual"
+# NAME
+
+river - dynamic tiling Wayland compositor
+
+# SYNOPSIS
+
+*river* [*-c* _shell_command_]
+
+# DESCRIPTION
+
+*river* is a dynamic tiling Wayland compositor inspired by dwm and
+bspwm based on wlroots and written in Zig.
+
+# OPTIONS
+
+*-c* _shell_command_
+	Run a shell command or give the path to a script that will be run
+	before river gets started. This is where you define keymaps using
+	*riverctl*(1) and autostart programs. Make sure the script is
+	executable.
+
+# CONFIGURATION
+
+View border width, gap sizes, and programs which should float are
+configured in _Config.zig_. Make your changes and recompile.
```

Comment:
This is no longer true for most configuration options.

Reply:
I will fix the outdated info in a separate PR.